rep-2005.rst

```diff
@@ -0,0 +1,245 @@
+Author: Brian Gerkey <gerkey@openrobotics.org>, Dirk Thomas <dthomas@openrobotics.org>
+Status: Active
+Type: Informational
+Content-Type: text/x-rst
+Created: 12-Jan-2020
+Post-History:
+
+
+Abstract
+========
+
+This REP describes a set of repositories which together form the standard library of ROS 2.
+
+
+Motivation
+==========
+
+Like ROS 1 before it, ROS 2 follows a federated development model.
+The open source code and other assets that make up the ROS 2 project are spread across multiple repositories, with different authors, maintainers, and contributors to each component.
+This federated model makes it easy for developers to share their contributions with the community while also retaining control over how those contributions are developed and released.
+As a result the community size and overall project scope can scale up naturally and efficiently.
+
+But in exchange, it can be difficult in a federated ecosystem for users and developers to know, of all the available ROS 2 packages, which are integral to the project.
+With the curated list below we aim to provide community members with guidance regarding which parts of the ROS 2 ecosystem are widely used, actively maintained, and of high quality.
+Furthermore, we encourage contributors to focus their efforts on items from this list.
+
+
+Scope
+=====
+
+The list below is curated to include repositories that are:
+
+* open source (not proprietary);
+* connected to the ROS 2 project (not general purpose software, except when included with ROS 2-specific bindings);
+* broadly usable (not narrowly tailored for uncommon use cases); and
+* evidently used by a nontrivial part of the community.
+
+We aim that all of these repositories are also:
+
+* actively maintained and following accepted software development processes, including code review and testing.
+
+
+Change process
+==============
+
+This content is maintained at GitHub in the [REP repo](https://github.com/ros-infrastructure/rep).
+To propose changes, such as adding or removing items, make a pull request to that repo.
+The ensuing public review and discussion will evaluate the change in light of the purpose and scope explained above, with the final merge decision being made by one of the REP maintainers.
+
+
+Content
+=======
+
+* Buildsystem, package index and tooling around
+
+  * `ament/ament_cmake <https://github.com/ament/ament_cmake>`_
+  * `ament/ament_index <https://github.com/ament/ament_index>`_
+  * `ament/ament_package <https://github.com/ament/ament_package>`_
+  * `ament/ament_lint <https://github.com/ament/ament_lint>`_
+  * `ros/ros_environment <https://github.com/ros/ros_environment>`_
+  * `ros-infrastructure/bloom <https://github.com/ros-infrastructure/bloom>`_
+  * `ros-infrastructure/catkin_pkg <https://github.com/ros-infrastructure/catkin_pkg>`_
+  * `ros-infrastructure/rosdep <https://github.com/ros-infrastructure/rosdep>`_
+  * `ros-infrastructure/rosdistro <https://github.com/ros-infrastructure/rosdistro>`_
```

Comment:
We have lot of activity around OpenEmbedded support for ROS1/ROS2 in https://github.com/ros/meta-ros and its actively maintained.
Can we add this repo to the list.

Reply:
@dirk-thomas  < Any opinion ?

Reply:
I would rather have others express their opinion since I shouldn't be the only one saying yes/no on those questions.